presto-main/src/main/java/com/facebook/presto/server/ServerInfoResource.java

```diff
@@ -92,4 +92,18 @@ public NodeState getServerState()
             return ACTIVE;
         }
     }
+
+    @GET
+    @Path("coordinator")
+    @Produces(TEXT_PLAIN)
+    public Response getServerCoordinator()
+    {
+        if (this.serverInfo.isCoordinator()) {
```

Comment:
@martint might have a different opinion, but I'm thinking the most RESTful thing to do here is return `204` (No Content) if a coordinator and `404` (Not Found) if not a coordinator. Would that work for you?